content/posts/testing-farm-triggering.md

```diff
@@ -0,0 +1,44 @@
+---
+title: "Running tests with builds from another PR"
+date: 2022-12-21
+weight: 93
+---
+
+Do you contribute to projects which depend on each other?
+Would you like to test changes spanning multiple repositories together before merging them to the main branch?
+Then look no further, Packit's new feature of the Testing Farm integration is what you are looking for!
+
+### How it works
+
+To enable such testing, there is no additional configuration required in your `packit.yaml`, the typical [Testing Farm configuration](/docs/testing-farm/) is sufficient.
+Once you open a pull request with some changes, tests are going to run as usual with all dependencies being installed based on the test definition, e.g. from Fedora's repositories.
+To trigger tests with builds from a pull request in another repository, add a comment to the pull request of the form:
+
+    /packit test <namespace>/<repo>#pr-id>
+
+Based on this comment, Testing Farm will first install the recent successful builds created by Packit in the given pull request and then run the tests.
+In order for this to work, there must be successful builds for the targets that you are running tests for.
+For example, if you are testing against Fedora 36, the pull request that you want to install builds from must contain a successful Fedora 36 build by Packit.
+
+Let's look at a simple example to demonstrate this feature better.
+The Packit CLI uses a library called `specfile` to modify rpm specfiles.
```

Comment:
```suggestion
The Packit CLI uses a library called `specfile` to modify RPM spec files.
```

Just to be consistent with specfile docs (even though I'm sure there are still plenty of inconsistencies regarding this in our projects).